extensions-contrib/kafka-emitter/src/main/java/org/apache/druid/emitter/kafka/KafkaEmitter.java

```diff
@@ -116,14 +123,21 @@ protected Producer<String, String> setKafkaProducer()
   @Override
   public void start()
   {
-    scheduler.schedule(this::sendMetricToKafka, 10, TimeUnit.SECONDS);
-    scheduler.schedule(this::sendAlertToKafka, 10, TimeUnit.SECONDS);
-    if (config.getRequestTopic() != null) {
+    if (config.getEventTypes().contains(KafkaEmitterConfig.EventType.METRICS)) {
```

Comment:
nit, you can factor out `config.getEventTypes()` into a variable to avoid the repetition and make the code more readable.
Similarly, you can add an import for `KafkaEmitterConfig.EventType`, so you can shorten the enum references to `EventType.XXXX`